templates/master.html

```diff
@@ -120,5 +121,14 @@
 <div id="content-body-container" style="margin: 25px;">
     {% block content_body %} {% endblock %}
 </div>
+{% if messages %}
+  <script>
+      {% for message in messages %}
+          $(document).ready(function () {
+              $('.toast').toast('show');
+          });
+      {% endfor %}
```

Comment:
Nice.
Did you test it with multiple messages?

Reply:
I tested it with 2 , it looks good, although they weren't long.